.github/workflows/preview.yml

```diff
@@ -22,6 +22,17 @@ jobs:
           files: posts/**
           json: "true"
 
+      - name: Create processed_files if not exist
```

Comment:
not sure whether this is really necessary or if it's not enough to just add the folder in a manual commit and document that this is required?

Reply:
I thought the same, but assuming someone wants to reuse this project, its nice that we generate this file, to avoid confused users that do not read the instructions.

Reply:
Ok, so we leave it manually.